src/ui/components/common/nav.rs

```diff
@@ -1,61 +1,155 @@
+      <div class="navbar-end">
+        <ul class="menu menu-horizontal flex-nowrap">
+          <li>
+            <A href="/search">
+              <span title=t!(i18n, nav_search)>
+                <Icon icon=Icon::from(ChIcon::ChSearch) class="h-6 w-6"/>
+              </span>
+            </A>
+          </li>
+          <li>
+            <A href="/login">{t!(i18n, nav_login)}</A>
+          </li>
+          <li>
+            <A href="/signup">{t!(i18n, nav_signup)}</A>
+          </li>
+          <li>
+            <A href="/inbox">
+              <span title=t!(i18n, nav_unread_messages)>
+                <Icon icon=Icon::from(ChIcon::ChBell) class="h-6 w-6"/>
+              </span>
+            </A>
+          </li>
+          <li>
+            <details>
+              <summary>"User name"</summary>
+              <ul>
+                <li>
+                  <A href="/u/jimmy90">{t!(i18n, nav_profile)}</A>
+                </li>
+                <li>
+                  <A href="/settings">{t!(i18n, nav_settings)}</A>
+                </li>
+                <li>
+                  <hr/>
+                </li>
+                <li>
+                  <ActionForm action=logout_form_action>
+                    <button type="submit">{t!(i18n, nav_logout)}</button>
+                  </ActionForm>
+                </li>
+              </ul>
+            </details>
+          </li>
+        </ul>
       </div>
-      <div class="navbar-end gap-3">
-        <button class="btn btn-ghost btn-circle">
-          <Icon icon=Icon::from(ChIcon::ChSearch) width="1.25rem" height="1.25rem"/>
-        </button>
-        <button class="btn btn-ghost btn-circle">
-          <div class="indicator">
-            <Icon icon=Icon::from(ChIcon::ChBell) width="1.25rem" height="1.25rem"/>
-            <span class="badge badge-xs badge-primary indicator-item"></span>
-          </div>
-        </button>
-        <A href="/login">"Login"</A>
-      </div>
-    </div>
+    </nav>
   }
 }
 
 #[component]
 pub fn BottomNav() -> impl IntoView {
+  let i18n = use_i18n();
+
   view! {
-    <footer class="sticky bottom-0">
-      <div class="btm-nav btm-nav-lg">
-        <A href="/" class="active">
-          // TODO put svg's here
-          <span class="btm-nav-label">"Home"</span>
-        </A>
-        <button>
-          <span class="btm-nav-label">"TODO 1"</span>
-        </button>
-        <button>
-          <span class="btm-nav-label">"TODO 2"</span>
-        </button>
+    <nav class="container navbar mx-auto">
+      <div class="navbar-start"></div>
+      <div class="navbar-end ">
+        <ul class="menu menu-horizontal flex-nowrap">
+          <li>
+            <a href="//github.com/LemmyNet/lemmy-ui-leptos/releases" class="text-md">
+              "f/e from env"
+            </a>
+          </li>
+          <li>
+            <a href="//github.com/LemmyNet/lemmy/releases" class="text-md">
+              "b/e from env"
+            </a>
+          </li>
```

Comment:
Are the double-slashes at the beginning of the hrefs intentional?

Reply:
without the double slash it get interpreted as a relative URL.